index.html

```diff
@@ -1044,6 +1110,7 @@ <h2>
              DOMString iarc_rating_id;
              USVString start_url;
              DisplayModeType display = "browser";
+             sequence&lt;DisplayModeType&gt; display_override;
```

Comment:
I just thought of a problem (which is not a problem right now, but will be as soon as we add a new display mode): we need a way to say for new display modes that they are allowed in `display_override` but not `display`.

We _either_ need to do this in the type system (so it is type-impossible to use, say, `"display": "tabbed"`, but you can use `"display_override": ["tabbed"]`), or have steps in the algorithm to fail if you use a "new" display mode in `display`. My preference is the former.

I think what we should do is have two enums, for now let's just call them "DisplayModeType" and "EnhancedDisplayModeType". The IDL should have:

```webidl
             DisplayModeType display = "browser";
             sequence<DisplayModeType or EnhancedDisplayModeType> display_override;
```

And never put a new value in `DisplayModeType`.

That allows you to use the new ones in `display_override` but not `display`, and we don't need to mention this in the algorithms at all. Does that sound good? We could also rename the old one to `LegacyDisplayModeType` or something, I don't really mind.

If we do that now, `EnhancedDisplayModeType` will be empty, so maybe we just add a note to do that in the future? Let's add a note just under the listing of `DisplayModeType`'s possible values, that no new values will be added to this, and future display modes will be added to a new enum that can only be used with `display_override` but not `display`, for forwards-compatibility reasons.

Reply:
Sound good, I'll add a note about this under the `DisplayModeType`'s possible values, saying:

> No new values will be added to this enum, and instead future display modes will be added to a new enum that can only be used with `display_override` but not `display`, for forwards-compatibility reasons. `display_override` will then be defined as `sequence<DisplayModeType or EnhancedDisplayModeType>`, assuming the new enum is named `EnhancedDisplayModeType`.

Reply:
Sounds good. Maybe reword it so it reads more easily: "will be added to a new enum, tentatively named `EnhancedDisplayModeType`, ..." and then you can remove the "assuming" qualifier to the last sentence.

Reply:
Based on internal implementation experience, it will be much easier to present this as the other option, where the algorithm fails for any 'new' display mode in `display`.  What do you think about that?


I made a note under the algorithm:
```
          In the future, new values added to {{DisplayModeType}} will not be
          added to the fallback chain, and the algorithm should treat the a
          {{WebAppManifest/display}} field set to a new value as {{browser}}.
```

And a note in DisplayModeType: